src/main/resources/db/changelog/changes/klimovych/insert-rentals-into-rental.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-rentals
+      author: nklimovych
+      changes:
+        - insert:
+            tableName: rentals
+            columns:
+              - column:
+                  name: rental_date
+                  valueDate: "2024-02-01"
+              - column:
+                  name: return_date
+                  valueDate: "2024-02-10"
+              - column:
+                  name: actual_return_date
+                  valueDate: "2024-02-09"
+              - column:
+                  name: car_id
+                  value: "1"
+              - column:
+                  name: user_id
+                  value: "1"
+        - insert:
+            tableName: rentals
+            columns:
+              - column:
+                  name: rental_date
+                  valueDate: "2024-04-01"
+              - column:
+                  name: return_date
+                  valueDate: "2024-04-05"
+              - column:
+                  name: actual_return_date
+                  valueDate: "2024-04-05"
+              - column:
+                  name: car_id
+                  value: "2"
+              - column:
+                  name: user_id
+                  value: "1"
```

Comment:
I think this is bad practice to hardcode here user and car id. Also, I believe we must use `preConditions` and `valueComputed`  in our liquibase changeset.
Let's imagine that someone deletes our test user or our test car with the following liquibase changes
then we will not be able to create these rentals and will get an error. For this, it is necessary to check whether these users and cars exist

Reply:
This insert changelog was removed